ztp/source-crs/StorageLV.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   logLevel: Normal
   managementState: Managed
   storageClassDevices:
-    - storageClassName: "fs-sc"
+    # The list of storage classes and associated devicePaths need to be specified like this example:
```

Comment:
why changing the storageClassName ? does changing the group-du-sno-ranGen.yaml not enough ? 
I'm just worry if someone use the StorageLV.yaml as it is, you force him to change the name. what do you think ?

Reply:
I think we are OK here. The StorageLV was added during 4.10 and has not yet been backported to 4.9

Reply:
The point of changing the source-crs is that no one should use the default value.  Storage layout is too user-specific and they must always put their config in their PGT.

Reply:
okay, if the back compatibility issue I guess its good then